Apps/frontend/src/views/StartStopButton.svelte

```diff
@@ -1,34 +1,24 @@
 <script>
 	import { createEventDispatcher } from 'svelte'
+	import Switch from "svelte-toggle";
 	export let channel_id;
 
 
 	const dispatch = createEventDispatcher()
 
-	let play = "Start ▶";
-	let stop = "Stop ■";
-	let symbol = channel_id + " " + stop;
 	let hasStarted = true;
+	let switchLabelOn = "Started " + channel_id;
```

Comment:
Not sure if the label "Started/Stopped" is needed anymore, as the toggle buttons already show the current state.